### PR TITLE
review for me

### PR DESCRIPTION
**Pull request by shestmintsev-kirill**

### CODE REVIEW

**Comment by alex-devdone** [comment]
Please fill the description of what was done in this PR (pull request)

**Comment by shestmintsev-kirill** [comment]
> Пожалуйста, заполните описание того, что было сделано в этом PR (pull request)
added editing and dynamic transition to course data

File: views/layouts/empty.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{{> head }}
```
Comment (alex-devdone) [inline views/layouts/empty.hbs:1]: It is a mistake or not?
```
{{>
```

Comment (shestmintsev-kirill) [reply views/layouts/empty.hbs:1]: No, this is the syntax Handlebars

Comment (alex-devdone) [reply views/layouts/empty.hbs:1]: Ok

**Pull request merged**